daemon/cmd/daemon.go

```diff
@@ -217,6 +217,8 @@ type Daemon struct {
 	cniConfigManager cni.CNIConfigManager
 
 	l2announcer *l2announcer.L2Announcer
+
+	settings cellSettings
```

Comment:
Given the name `settings` is quite generic, it's not obvious what is represents. Can you add a godoc?